src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -2887,6 +2889,24 @@
 
 (string templateFileName, Dictionary<string, string> templateData)[] SveInputs = new []
 {
+    ("SveSimpleVecOpTest.template",       new Dictionary<string, string> { ["TestName"] = "Sve_Abs_float",                                                                                         ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "Abs",                                                                  ["RetVectorType"] = "Vector",    ["RetBaseType"] = "Single",  ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "Single",                                                                                                                           ["LargestVectorSize"] = "8",  ["NextValueOp1"] = "-TestLibrary.Generator.GetSingle()",                                                                                                                                                     ["ValidateIterResult"] = "Helpers.Abs(firstOp[i]) != result[i]"}),
```

Comment:
Why is `LargestVectorSize == 8`, presumably it should be more like `256` which is the largest supported SVE vector length?

Or at worst it should be `16` (the minimum SVE vector length), `32` (the largest size in server grade hardware), or `64` (the largest size in any SVE implementation to date, notably only in a supercomputer).

Reply:
yes, I think the same as I commented in https://github.com/dotnet/runtime/issues/100366#issuecomment-2024559892.